var/spack/repos/builtin/packages/aws-parallelcluster/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class AwsParallelcluster(PythonPackage):
```

Comment:
I opted for `aws-parallelcluster` instead of `py-aws-parallelcluster` as it seems like this package is primarily used from the CLI. This is similar to why `awscli` isn't called `py-awscli`.